lnbits/static/js/components/lnbits-funding-sources.js

```diff
@@ -1,6 +1,14 @@
 Vue.component('lnbits-funding-sources', {
   mixins: [windowMixin],
   props: ['form-data', 'allowed-funding-sources'],
+  methods: {
+    getFundingSourceLabel(item) {
```

Comment:
why is this needed?
what was being displayed without it?

Reply:
Without this, the [0] value is being displayed e.g. "CoreLightningWallet" this changes the drop down labels to:

CoreLightningWallet > Core Lightning
CoreLightningRestWallet > Core Lightning Rest
LndWallet > Lightning Network Daemon (LND)

etc